kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -481,7 +481,10 @@ define([
             handleJobStatus(self, message) {
                 const job = this.model.getItem('exec.jobState');
                 if (message[job.job_id]) {
-                    const jobState = message[job.job_id].jobState;
+                    const { outputWidgetInfo, jobState } = message[job.job_id];
+                    if (outputWidgetInfo) {
+                        self.model.setItem('exec.outputWidgetInfo', outputWidgetInfo);
+                    }
```

Comment:
make sure the job manager saves the output widget info when it's dealing with single app cells